README.md

```diff
@@ -59,6 +59,24 @@ A simple helper to perform async application initialization for the generic host
     }
     ```
 
+    you can also pass a `CancellationToken` in order to propagate notifications to cancel the initialization if needed.<br />
+    In the following example, the initialization will be cancelled 
+    ```csharp
+    public static async Task Main(string[] args)
+    {
+        CancellationToken cancellationTokenSource = new CancellationTokenSource();
```

Comment:
This code wouldn't compile (you can't assign a `CancellationTokenSource` to a `CancellationToken`). Consider using `var` to keep the code more concise.
Also, the `CancellationTokenSource` should be disposed after use. You can just add `using ` before the declaration.